dsc/src/util.rs

```diff
@@ -32,6 +32,7 @@ pub const EXIT_DSC_ERROR: i32 = 2;
 pub const EXIT_JSON_ERROR: i32 = 3;
 pub const EXIT_INVALID_INPUT: i32 = 4;
 pub const EXIT_VALIDATION_FAILED: i32 = 5;
+pub const EXIT_CTRL_C: i32 = 6;
```

Comment:
After this PR lands, I'm going to update the CLI docs to include exit codes for DSC itself.

Reply:
Actually, would it be possible to get short docs strings for these constants? That would make documenting them in CLI docs easier. Right now I'm trying to track down how they're used throughout the code.

Reply:
We can add them in a separate PR.  Can probably put them in the `.dsc.resource.json` files.  Created https://github.com/PowerShell/DSC/issues/181